### PR TITLE
doc: add documentation on review apps interactions

### LINKED ISSUE #305: Add review app interaction events
As specified [here](https://www.notion.so/scalingooriginal/Create-a-new-event-4bf66e5e33ac4cada338ebb94c1f8a7a?pvs=4#4e38ec0196fe4b7687c6ee023ce67f6a)

### PR DESCRIPTION
**Pull request by yohann-bacha**
fixes #305

### CODE REVIEW

File: _scalingo_api/events.md [hunk 1]
```diff
@@ -1281,6 +1281,101 @@ Example object:
...
+
+_When:_ Each time a Review App is destroyed
+`type=destroy_review_app`
+
+{:.table}
+| field                | type    | description                                               |
+| -------------------- | ------- | --------------------------------------------------------- |
+| review_app_name      | string  | Name of the Review App                                    |
+| source_repo_name     | string  | Name of the repository originating the Pull/Merge Request |
+| source_repo_url      | string  | URL of the repository originating the Pull/Merge Request  |
```
Comment (guillaume-sc) [inline _scalingo_api/events.md:1342]: See my comments above regarding the verb "to originate".

File: _scalingo_api/events.md [hunk 2]
```diff
@@ -1281,6 +1281,101 @@ Example object:
...
+| pr_name              | string  | Pull/Merge Request name                                   |
+| pr_number            | integer | Pull/Merge Request number                                 |
+| pr_url               | string  | Pull/Merge Request URL                                    |
+| pr_comes_from_a_fork | boolean | If the Pull/Merge Request comes from a fork repository    |
```
Comment (guillaume-sc) [inline _scalingo_api/events.md:1346]: See my comment above regarding an alternative text.

**Review by guillaume-sc — CHANGES_REQUESTED** [review]
Few comments regarding English wording.

File: _scalingo_api/events.md [hunk 3]
```diff
@@ -1281,6 +1281,101 @@ Example object:
...
+_When:_ Each time a Review App is created
+`type=create_review_app`
+
+{:.table}
+| field                | type    | description                                               |
+| -------------------- | ------- | --------------------------------------------------------- |
+| review_app_name      | string  | Name of the Review App                                    |
+| review_app_url       | string  | Dashboard URL of the Review App                           |
+| source_repo_name     | string  | Name of the repository originating the Pull/Merge Request |
+| source_repo_url      | string  | URL of the repository originating the Pull/Merge Request  |
```
[the remark this answers is not in the document]
Comment (yohann-bacha) [reply _scalingo_api/events.md:1295]: Yup, fixing this

File: _scalingo_api/events.md [hunk 4]
```diff
@@ -1281,6 +1281,101 @@ Example object:
...
+| pr_name              | string  | Pull/Merge Request name                                   |
+| pr_number            | integer | Pull/Merge Request number                                 |
+| pr_url               | string  | Pull/Merge Request URL                                    |
+| pr_comes_from_a_fork | boolean | If the Pull/Merge Request comes from a fork repository    |
```
[the remark this answers is not in the document]
Comment (yohann-bacha) [reply _scalingo_api/events.md:1299]: Way better, fixing this

File: _scalingo_api/events.md (hunk 1 above)
Comment (yohann-bacha) [reply _scalingo_api/events.md:1342]: Same, fixing this

File: _scalingo_api/events.md (hunk 2 above)
Comment (yohann-bacha) [reply _scalingo_api/events.md:1346]: Fixing this

**Review by guillaume-sc — APPROVED** [review]

**Pull request merged**